css/about.css

```diff
@@ -0,0 +1,221 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;500;600;700&display=swap");
+
+:root {
+  --color-white: #fff;
+  --color-header: #272a31;
+  --font-family: "Lato", sans-serif;
+  --color-primary: #ff5a5f;
+}
+
+.tonnys-logo {
+  width: 50% !important;
+  height: 50% !important;
```

Comment:
- [ ] Please don't use !important in CSS. important flag are applied to the same element, the declaration with a greater specificity is applied. Using ! important to override specificity is considered a bad practice and should be avoided for this purpose.